vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -17,9 +17,10 @@ NODE_DRAIN_TIMEOUT: "{{ lookup('env', 'NODE_DRAIN_TIMEOUT') | default('0s', true
 BMOPATH: "{{ lookup('env', 'BMOPATH') }}"
 IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
-KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.19.3', true) }}"
-
-KUBERNETES_BINARIES_VERSION: "{{ lookup('env', 'KUBERNETES_BINARIES_VERSION') | default(lookup('env', 'KUBERNETES_VERSION') | default('v1.18.0', true), true) }}"
+# (FIXME) Add a lookup info front of KUBERNETES_VERSION and 
+# KUBERNETES_BINARIES_VERSION vars to inherit values from lib/common.sh
+KUBERNETES_VERSION: "v1.19.3"
+KUBERNETES_BINARIES_VERSION: "v1.19.3"
```

Comment:
Why version 1.19.3 is still used here?

Reply:
That's for the target cluster. Right now I am trying with the source cluster first.